rest-dashboard.go

```diff
@@ -165,11 +165,12 @@ func (r *Client) SearchDashboards(query string, starred bool, tags ...string) ([
 // may be only loaded with HTTP API but not created or updated.
 //
 // Reflects POST /api/dashboards/db API call.
-func (r *Client) SetDashboard(board Board, overwrite bool) (StatusMessage, error) {
+func (r *Client) SetDashboard(board Board, folderId int, overwrite bool) (StatusMessage, error) {
 	var (
 		isBoardFromDB bool
 		newBoard      struct {
 			Dashboard Board `json:"dashboard"`
+			FolderID  int   `json:"folderId"`
```

Comment:
Can we add some tests for this to be sure? (:

Reply:
Sure.

Reply:
Thanks!